core/src/services/s3/core.rs

```diff
@@ -517,25 +515,23 @@ impl S3Core {
     pub async fn s3_initiate_multipart_upload(
         &self,
         path: &str,
-        content_type: Option<&str>,
-        content_disposition: Option<&str>,
-        cache_control: Option<&str>,
+        args: &OpWrite,
```

Comment:
Hi, please also check the place where uses those functions.

Reply:
i see this
https://github.com/apache/incubator-opendal/blob/d30ff84d8c718797106dfaf90e76a5ae3ef110d3/core/src/services/s3/core.rs#L530-L540

Reply:
Please check the place where called `s3_put_object_request`, they should also be updated.

`cargo check` will help point out the detailed position.

Reply:
already fixed